src/sentry/migrations/0734_prepare_sunset_appstore_connect_integration.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 5.0.6 on 2024-07-08 17:27
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0733_relocation_provenance"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[migrations.DeleteModel(name="LatestAppConnectBuildsCheck")],
+            database_operations=[],
+        ),
+        migrations.SeparateDatabaseAndState(
+            state_operations=[migrations.DeleteModel(name="AppConnectBuild")],
+            database_operations=[],
+        ),
```

Comment:
You can combine these into one `SeparateDatabaseAndState` operation if you want. It can also be left this way.

Reply:
It's better if we merge it. I updated the pull-request. Thanks for the review.